Task Liverpool/src/main/java/objects/Product.java

```diff
@@ -0,0 +1,52 @@
+package objects;
```

Comment:
It's more common to use other package names like: model or dao

Reply:
Alright, I changed the package name to dao.